src/Collisions/collider.ts

```diff
@@ -260,6 +260,7 @@ export class Collider {
         if (!embeddedInPlane) {
             this._basePoint.subtractToRef(trianglePlane.normal, this._planeIntersectionPoint);
             this._velocity.scaleToRef(t0, this._tempVector);
+            this._velocitySquaredLength = this._velocity.lengthSquared();

```

Comment:
Is this line necessary? It is already done in `initialize` (I guess this function is always called before `_testTriangle`?) and `_velocity` is not modified anywhere in the class (and is private, so can't be modified by the outside), so the value computed in `initialize` should still be the right one.

Reply:
Good catch! removed now